### DISCUSSION TITLE
no emails to callbacks

### DISCUSSION THREAD

**Discussion by Falconcrest** [nzymedefense/nzyme#520] [Questions] [answered]
In addition to the configuration in nzyme.conf, are there any other things required on the raspberry pi so that the emails are sent? I assume that my entries in nzyme.conf are correct ...

**Comment by lennartkoopmann** [comment]
Nzyme is using SMTP directly and there is nothing else required locally. Feel free to post your configuration (please redact credentials) and your nzyme log file.

**Comment by Falconcrest** [comment]
Here comes my configuration:

```
alerting {
    # Notifications and callbacks for triggered alerts.
    callbacks: [
      {
        type: email
        enabled: true

        # One of: SMTP, SMTPS or SMTP_TLS
        transport_strategy: SMTP_TLS

        host: smtp.ionos.de
        port: 465
        username: "username@mail.de"
        password: "pwd"

        from: "nzyme <admin@mail.de>"
        subject_prefix: "[NZYME]"

        recipients: [
          "Some body <somebody@mail.de>"          
        ]
      }
```

SMTPS and Port 587 does not work too.

**Reply by lennartkoopmann** [reply]
(I edited your response for better code formatting / readability)

That looks correct. Can you post your nzyme log file? It should include an error that will tell us more.

**Reply by WhoCares1337** [reply]
I had problems getting email notifications to work as well. Maybe you can implement a "send testmail" button? I had to restart nzyme a dozen times and the entries in the log weren't very helpfull ( I'm afraid I don't have them anymore)

**Reply by lennartkoopmann** [reply]
Good idea! I created a feature request here: #580 

If you find the logs, feel free to post them and I'm happy to see if the error messages can be improved.

**Comment by JohnDoeIsKIng** [comment, marked as answer]
For me SMTPS works with port 465.
Perhaps one has to authenticate at the mail server for sending mails.
Greets

JD.
